index.html

```diff
@@ -12,6 +12,66 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!--Navbar-->
+    <div class="container">
+        <div class="logo">
+          <img src="/img/karma-logo.svg" alt="logo">
+          </div>
+        <nav>
+          <ul>
+          <li><b>Meet Karma</b></li>
+          <li>How it Works</li>
+          <li>Store</li>
+          <li>Blog</li>
+          <li>Help</li>
+          <li>Login</li>
+          </ul>
+        </nav>
+    </div>
+
+        <div class="bckimg">
+            <br>
+            <br>
+            <br>
+            <br>
+            <p>Introducing Karma</p>
+            <p>Bring WiFi with you,everywhere you go.</p>
+            <button class="btn">Learn More</button>
+        </div>
+        <br>
+        <br>
+        <!--Services-->
+        <div class="services">
+            <p>Everyone needs a little Karma.</p>
+        </div>
+
+            <div class="column">
+                <img class="img" src="/img/icon-devices.svg" alt="Jane">
+                  <p>Internet for all devices</p>
+            </div>
+          
+            <div class="column">
+                <img class="img" src="/img/icon-coffee.svg" alt="Mike">
+                  <p>Boost your productivity</p>
+            </div>
+            
+            <div class="column">
+                <img class="img" src="/img/icon-refill.svg" alt="John">
```

Comment:
Who are Jane, Mike and John? 😄  Remember to use descriptive text inside the alt attribute.